lib/evss/disability_compensation_form/dvp/configuration.rb

```diff
@@ -7,7 +7,7 @@ module Dvp
       # points to the Digital Veterans Platform
       class Configuration < EVSS::DisabilityCompensationForm::Configuration
         def base_path
-          "#{Settings.evss.dvp.url}/#{Settings.evss.dvp.service_name}/rest/form526/v2"
+          "#{Settings.dvp.url}/#{Settings.evss.service_name}/rest/form526/v2"
```

Comment:
I see what `Settings.evss.service_name` connects to in settings.yaml:
```
evss:
  prefill: true
  url: https://csraciapp6.evss.srarad.com
  service_name: "wss-form526-services-web"     <--------
  alternate_service_name: "wss-form526-services-web-v2"
  cert_path: ~
  key_path: ~
  root_cert_path: ~
  versions:
    claims: 3.6
    common: 11.6
    documents: 3.7
```
But I don't see what `Settings.dvp.url` connects to (nor do I see what `Settings.evss.dvp.url` used to connect to 🤔)

Reply:
but the environment specific `.local.yml` files get added during the builds, correct? For example, a Firefly engineer added the original dvp url, [shown here](https://github.com/department-of-veterans-affairs/devops/blob/0a3b4aaca617a2ffa7cac94f26777adfda77c173/ansible/deployment/config/vets-api/dev-settings.local.yml.j2#L55)... and the dev console is showing such. Or is this removed from the build now? I remember a slack message recently saying that maybe only prod is getting the settings file still?
```
irb(main):001:0> Settings.evss.dvp.url
=> "https://blue.qa.lighthouse.va.gov"
```

and if that's the case, [this PR](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/1921/files) would make the needed changes to dev, correct? I'm not familiar with the manifest repo, so correct me if I'm wrong.

Reply:
i was replacing a reference to a setting that we changed in vets-api settings.yml

Reply:
> but the environment specific .local.yml files get added during the builds, correct?

no, not anymore. Everything in [this section of values.yaml](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/prod/values.yaml#L1537-L1541) gets mounted to `/app/config/settings.local.yml` in the pod. So the reason you're able to see `Settings.evss.dvp.url` (in a dev pod) is because of the settings defined in the [dev values.yaml file](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/dev/values.yaml#L1658). Here's a portion of the settings.local.yml file in a vets-api-prod pod:
```
nonroot@vets-api-web-5b568b877-4m45b:~$ tail -20 /app/config/settings.local.yml
    aws_access_key_id: <%= ENV['DHP_AWS_ACCESS_KEY_ID'] %>
    aws_secret_access_key: <%= ENV['DHP_AWS_SECRET_ACCESS_KEY'] %>
    region: <%= ENV['DHP_AWS_REGION'] %>
    bucket: <%= ENV['DHP_TOKEN_STORAGE_BUCKET'] %>

dgi:
  jwt:
    public_key_path: /srv/vets-api/secret/production/dgi-prod.crt
    private_key_path: /srv/vets-api/secret/dgi-prod.key
  vets:
    url: <%= ENV['DGI_VETS_URL'] %>

lgy:
  base_url: <%= ENV['LGY_BASE_URL'] %>
  app_id: VAGOVSERVICE
  api_key: <%= ENV['LGY_API_KEY'] %>
  mock_coe: false

dogstatsd:
  enabled: true
nonroot@vets-api-web-5b568b877-4m45b:~$
```

Reply:
Does that make sense? 
With EKS, the settings get loaded in from the `settings-configmap` in values.yaml in the manifest repo. `settings.yml` in vets-api has default values but then the values per environment will get loaded in from the configmap. It's best practice to keep vets-api settings.yml and the configmaps inline even if its blank. So settings.yml needs to have something like this
```
evss:
  dvp:
    url: ~
```